backend/danswer/server/features/persona/api.py

```diff
@@ -239,11 +239,14 @@ def get_persona(
     persona_id: int,
     user: User | None = Depends(current_user),
     db_session: Session = Depends(get_session),
+    include_non_owned: bool = Query(
```

Comment:
would slightly prefer to not offer that (e.g. just use a default icon if the user doesn't have access to the persona). Also, if we do keep this it should likely be something like `include_non_accessible`, since it already returns those that the user doesn't own

Reply:
Agree on all fronts. Didn't mean to take this out of drafts. Was still testing to see if I could understand why the original issue was occurring.